Library/Homebrew/utils/ruby.sh

```diff
@@ -45,15 +50,24 @@ EOS
       then
         HOMEBREW_RUBY_PATH="/System/Library/Frameworks/Ruby.framework/Versions/Current/usr/bin/ruby"
       else
-        HOMEBREW_RUBY_PATH="$(type -P ruby)"
+        IFS=$'\n'
+        for ruby_exec in $(which -a ruby) $(PATH=$HOMEBREW_PATH which -a ruby)
+        do
+          if [[ $(test-ruby "$ruby_exec") == "true" ]]; then
+            HOMEBREW_RUBY_PATH=$ruby_exec
+            break
+          fi
+        done
+        IFS=$' \t\n'
```

Comment:
Is it worth setting this back to whatever it was before e.g. `OLD_IFS` or something?

Reply:
`$' \t\n'` is the default value for `IFS`:

>  IFS    The Internal Field Separator that is used  for  word  splitting after expansion  and  to  split lines into words with the read builtin command.   The  default  value  is  ``\<space\>\<tab\>\<new-line\>''.

I'm restoring it back to its normal value after setting it to `$'\n'` which causes word splitting to happen on new lines only.

Reply:
Yup but now to understand the code I have to know that this is the default value whereas if you used `OLD_IFS` I'd know why it was being set back to this 😁

Reply:
> to understand the code I have to know that this is the default value

I can add a comment explaining/reminding that `$' \t\n'` is the default value.

Reply:
Works for me 👍